docs/docs/components/guidelines/README.md

```diff
@@ -131,20 +131,16 @@ run the following commands at the root of the project:
 
 To create and submit a new or updated component to the Pipedream registry:
 
-1. Fork or branch the `pipedreamhq/pipedream` repo
-2. Develop and test a new or updated component that conforms to the guidelines
-   in the document
-3. Create a PR for the Pipedream team to review and post a message in our
```

Comment:
Can you actually keep steps 3, 4 and 5? I would add them between steps 8 and 9 of your list.